### PR TITLE
Add in dispatcher and handlers for slash commands

### PR DESCRIPTION
**Pull request by rogeruiz**
## Background

This work is tied to the [[AWS] Create GitHub Actions configuration for Terraform initialization of pull requests][asana] :lock:.

[asana]: https://app.asana.com/0/1181500399442529/1200106415052877/f

## How Has This Been Tested

~~This will be tested in this pull request with Slash commands issued by maintainers.~~

```sh
act repository_dispatch -j tf_init -s "TF_API_TOKEN=<REDACTED>"

# redacted output for brevity...
[Pull Request Handler/Run tf-init]   ⚙  ::set-output:: stderr=
[Pull Request Handler/Run tf-init]   ⚙  ::set-output:: exitcode=0
[Pull Request Handler/Run tf-init]   ✅  Success - Terraform Init

```

### Test Configuration

* Terraform Version: 0.14.7
* Any additional relevant variables:

## This PR makes me feel

George Lopez behind a laptop saying That's Terrifying

### CODE REVIEW

File: .github/workflows/dispatcher.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  slash_command_dispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Slash Command Dispatch
+        uses: peter-evans/slash-command-dispatch@v2
+        with:
+          token: ${{ secrets.PAT }}
+          commands: |
+            init
```
Comment (aaron-lane) [inline .github/workflows/dispatcher.yml:15]: This command should be "test" as we want to kick off the entire workflow with a single command; also, it must match one of the `repository_dispatch.types` defined in `handler.yml` when combined with the `event-type-suffix`.

```suggestion
            test
```

**Review by aaron-lane — CHANGES_REQUESTED** [review]
I'm excited to see this part come together!

By the by, I don't think GitHub Actions will process this new configuration until the pull request is merged, so we may have to iterate.

**Comment by rogeruiz** [comment]
@aaron-lane I'm trying to leverage the `act` tool to iterate on this locally. but you might be right on needing this merged into the default branch.

File: .github/workflows/dispatcher.yml (hunk 1 above)
Comment (rogeruiz) [reply .github/workflows/dispatcher.yml]: I'm not sure I understand. running `/ test` will kick off everything? or are we going for multiple slash commands?

Comment (aaron-lane) [reply .github/workflows/dispatcher.yml]: Right, `/test` will kick off everything.

**Review by aaron-lane — APPROVED** [review]

**Comment by rogeruiz** [comment]
I ran this locally with `act` using the following command and had it successfully initialize.


```sh
act repository_dispatch -j tf_init -s "TF_API_TOKEN=<REDACTED>"

# redacted output for brevity...
[Pull Request Handler/Run tf-init]   ⚙  ::set-output:: stderr=
[Pull Request Handler/Run tf-init]   ⚙  ::set-output:: exitcode=0
[Pull Request Handler/Run tf-init]   ✅  Success - Terraform Init

```

**Review by aaron-lane — APPROVED** [review]
LGTM 🌬️

**Pull request merged**